.travis.yml

```diff
@@ -28,6 +28,8 @@ matrix:
     - php: 5.5
       env: SYMFONY_VERSION="2.6.*"
     - php: 5.5
+      env: SYMFONY_VERSION="2.7.*" PHPUNIT_BRIDGE=true SYMFONY__DB__DRIVER="pdo_mysql"
```

Comment:
do you actually need to run tests on MySQL ? this bundle only uses basic ORM features AFAICT, which are already tested on all platforms by the ORM team. this looks like you are testing the ORM itself here